src/modules/sensors/als.c

```diff
@@ -2,8 +2,8 @@
 #include <udev.h>
 
 #define ALS_NAME            "Als"
-#define ALS_ILL_MAX         500
-#define ALS_ILL_MIN         22
+#define ALS_ILL_MAX         100000 // Direct Sunlight
+#define ALS_ILL_MIN         20 // Dark Room
```

Comment:
Why not using 1 as default ALS_ILL_MIN?
Just a curiosity!

Reply:
Moreover, can you drop a comment with link to relative doc here (https://docs.microsoft.com/en-us/windows/win32/sensorsapi/understanding-and-interpreting-lux-values)
Thanks!

Reply:
I chose 20 instead of 1, since a lux of 1 would be essentially if the sensor was in a pitch black cave. Though upon further experimentation with my own ALS, it seems it's not sensitive enough at that range to begin with (i could only get it to 315 lux before any darker would plummet it to 0 lux). So perhaps an even higher minimum value would be better depending on if we want the default behavior to be the backlight minimizing before the ALS craps out.

Reply:
As pointed above, values returned by Clightd are not directly used as a backlight level, but Clight has a backlight curve to correlate them.
It is fine to use 1 as min value, eventually returning 0.0 ambient brightness for very very very dark environments.